src/core/webidl.js

```diff
@@ -285,6 +270,24 @@ function renderWebIDL(idlElement) {
     }
     registerDefinition(elem, [title]);
   });
+  // cross reference
+  const closestCite = idlElement.closest("[data-cite], body");
+  const { dataset } = closestCite;
+  if (!dataset.cite) {
+    dataset.cite = "WebIDL";
+    return;
+  }
+  const hasWebIDL = dataset.cite
+    .split(/\s+/)
+    .map(item => item.toLowerCase())
+    .includes("webidl");
+
+  if (hasWebIDL) return;
+
+  dataset.cite = ["WebIDL"]
+    .concat(dataset.cite.split(/\s+/))
```

Comment:
Might be better to do `const cites = dataset.cite.split(/\s+/)` rather than repeating this.

Reply:
true. Will fix locally.